speech.yaml

```diff
@@ -209,12 +209,13 @@ components:
             - '0'
             - '1'
         substitutions:
-          type: string
+          type: object
+          additionalProperties:
+            type: string
           description:
-            A set of substitutions to apply to the transcribed text,
-            encoded in a stringified-JSON format of key/value
+            A set of substitutions to apply to the transcribed text, key/value
```

Comment:
Minor text tweak:
```
A set of substitutions to apply to the transcribed text, in key/value format.
```

Is there anything we want to call out about the key or value content, such as use of regex or other things?